dev_readme.md

```diff
@@ -99,20 +99,45 @@ Note: Please note that certain aspects of the website may not be fully functiona
   >$ git clone https://github.com/YOUR-USERNAME/AutSPACEs
   7. Press enter. Your local clone will be created. 
 
-### Extra Tips
-
-*If you aren't sure how to get to the root repository, you can find it more easily by following these steps:* 
-
-1. Type `git config --global alias.root 'rev-parse --show-toplevel` into your terminal. This will create a simpler command to take you to the root repository. 
-2. Type `cd $(git root)` into your terminal. This will take you to the root of the repository you are in. 
-
-#### Accessing the database
+### Accessing the database
 
 To access the PostgreSQL database:
 
 1. Run `docker exec -it autspaces_db_1 bash`
 2. In the subsequent terminal, run `psql --username=autspaces`
 
-### Docker overview 
+### Accessing the Django shell
+
+Django provides a way to load a Python shell that has pre-loaded all Django settings. This allows loading models, executing database queries etc. for testing purposes. To access the shell run the following command while the `./docker-run.sh` is still active in a different terminal:
```

Comment:
Yeah, I feel it would be best to keep it at each command as otherwise people are likely to skip to the one they want and wonder why it doesn't work :D